rosidl_typesupport_opensplice_cpp/rosidl_typesupport_opensplice_cpp/__init__.py

```diff
@@ -13,11 +13,31 @@
 # limitations under the License.
 
 import os
+import re
 import subprocess
 
 from rosidl_cmake import generate_files
 
 
+def check_idlpp_supports_include_namespaces(idl_pp):
+    version = 0
+    build = 0
+
+    r = subprocess.getstatusoutput(idl_pp + ' -v')
```

Comment:
@dirk-thomas nit: if we do not care about the status, and since this function [is tagged as legacy](https://docs.python.org/3.6/library/subprocess.html#legacy-shell-invocation-functions) in Python3 documentation, consider using [`subprocess.check_output()`](https://docs.python.org/3.6/library/subprocess.html#subprocess.check_output) instead.

Reply:
Yeah, the patch is based on an external contribution. I completely agree: 5c198963f60894970d17da0d9e132f63717e87b0